src/ui/descendents.rs

```diff
@@ -64,6 +115,7 @@ fn test_get_descendents_depth_2() {
 }
 
 // parses /proc/<pid>/status format
+#[cfg(target_os = "linux")]
```

Comment:
Should these be `cfg(unix)` instead?

Reply:
On macos, we use `libproc` to get process children, so this test is linux-specific.

> > Alright. I figured this out.
> > It seems like we cannot profile 32-bit Ruby on 64-bit Windows.
> > That holds true even apart from this request. Should we document this behavior?
> > @liaden and @jvns could you take a glance at the PR?
> 
> Can we detect this behavior when the user attempts to profile a 32 bit ruby and surface a helpful error message?